spec/client_server_spec.rb

```diff
@@ -31,25 +38,39 @@ def recording_server(replies = {})
       }
     end
 
-    it 'should issue select command before succeeding connection' do
+    it 'should issue ping command before succeeding connection' do
```

Comment:
This test is actually checking for `ping` and `select` 👍

Reply:
true! i changed the description